src/gui/search/searchwidget.cpp

```diff
@@ -74,13 +74,18 @@ SearchWidget::SearchWidget(MainWindow *mainWindow)
     , m_mainWindow(mainWindow)
     , m_isNewQueryString(false)
     , m_noSearchResults(true)
-    , m_nbSearchResults(0)
 {
     setupUi(this);
 
     m_searchPattern = new LineEdit(this);
     searchBarLayout->insertWidget(0, m_searchPattern);
     connect(m_searchPattern, SIGNAL(returnPressed()), searchButton, SLOT(click()));
+    m_searchPattern->setToolTip(tr("<html><head/><body><p>A phrase to search for.<br>"
+          "Spaces in the search terms may be protected by double quotes.</p>"
+          "<p>Example:<br>"
+          "<b>foo bar</b>: search for <b>foo</b> and <b>bar</b><br>"
+          "<b>&quot;foo bar&quot;</b>: search for <b>foo bar</b><br>"
+          "</p></body></html>"));
```

Comment:
Unfortunately I think you need to break this up in multiple sub strings for 2 reasons.
1. Any change in the html code will break all translations
2. Some translator might forget an html tag and make the whole thing render weirdly. Or worse insert its own tags.